pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1715,6 +1715,9 @@ self: super: {
   # Issue reported upstream, no bug tracker url yet.
   darcs = doJailbreak super.darcs;
 
+  # Too strict version bounds on ansi-terminal;
+  colourista = doJailbreak super.colourista;
```

Comment:
Could you report this upstream, and link the issue here?

Reply:
There is already an upstream commit that fixes the issue https://github.com/kowainik/colourista/commit/15ace92105b56eba4ea3717bd55f733afe5be401. Not sure how to get that into package though.

Reply:
You can either use `appendPatch` or just add a “remove at release after <current release>” to the comment.

Reply:
Example:

<https://github.com/NixOS/nixpkgs/blob/0c9f9676c7ccadde3cd05f399898bbfe7d5a0831/pkgs/development/haskell-modules/configuration-common.nix#L713-L718>

Reply:
Converted it to the patch.